epoxy-processor/src/main/java/com/airbnb/epoxy/DataBindingAttributeInfo.java

```diff
@@ -1,4 +1,22 @@
 package com.airbnb.epoxy;
 
+import com.squareup.javapoet.TypeName;
+
+import javax.lang.model.element.Element;
+
 class DataBindingAttributeInfo extends AttributeInfo {
+
+  public DataBindingAttributeInfo(DataBindingModelInfo modelInfo, Element dataBindingElement) {
+    name = ProcessorUtils.decapitalizeFirstLetter(dataBindingElement.getSimpleName().toString());
+    typeName = TypeName.get(dataBindingElement.asType());
+    typeMirror = dataBindingElement.asType();
+    modelName = modelInfo.getGeneratedName().simpleName();
+    modelPackageName = modelInfo.getGeneratedName().packageName();
+    useInHash = true;
```

Comment:
for `useInHash` we probably need to exclude certain things, like ClickListeners. I have the same problem and a TODO with the litho models and models generated from Views. we can leave this as is and solve them all at once later on, but let me know if you have ideas.